architecture.md

```diff
@@ -12,27 +12,27 @@ babelfish **server** in a very specific runtime.
 
 ## Language Drivers
 
-A language driver is a program, containerized with Docker, that takes file parsing
+A language driver is a program, containerized with [Docker](https://www.docker.com/get-docker), that takes file parsing
 requests with source code and replies with an [UAST](./uast/specification.md)
 representation of the provided source code or an error message.
 
-Our implementations of language drivers are composed of two pieces: a code parser,
-that can be written in any language (usually the source language) and an AST to
-UAST normalizer written in Go. In the case of code parsers written in Go or other
-languages producing linkable object files or shared library, the code parser and UAST
-normalizer might be combined by dynamic or static linking without a separate
+Our implementations of language drivers are composed of two pieces: a *code parser*,
+that can be written in any language (usually the source language) and an
+*AST*to*UAST*normalizer* written in Go. In the case of code parsers written in Go or other
+languages producing linkable object files or shared library, the code parser and
+UAST normalizer might be combined by dynamic or static linking without a separate
 runnable component.
 
 The entry point of the container is the Go UAST normalizer, which communicates
 internally with the language-specific code parser.
 
-## Server
+## Daemon
 
-The server is the higher level component of the architecture managing
-client requests (done via [gRPC](http://www.grpc.io) using [a simple 
-protocol](driver/protocol.md)) with the containerized
-language drivers which it orchestate to keep warm instances, pools and handing
-any other orchestation needs.
+The *bblfshd* is the higher level component of the architecture managing
+client requests (done via [gRPC](http://www.grpc.io) using [a simple
+protocol](driver/protocol.md)) with the containerized language drivers which it
```

Comment:
`driver/protocol.md` doesn't exist anymore.

Reply:
the links was wrong, it should point to the main protocol